packages/@aws-cdk/aws-iam/lib/policy-document.ts

```diff
@@ -1,6 +1,8 @@
 import * as cdk from '@aws-cdk/core';
 import * as cxapi from '@aws-cdk/cx-api';
+import { IConstruct } from 'constructs';
 import { PolicyStatement } from './policy-statement';
+import { mergeStatements } from './private/merge-statements';
```

Comment:
[nit pick] The capitalization breaks the expected pattern.

Reply:
Oh aha, is this what you meant.

* `PolicyStatement` is a class
* `mergeStatements` is a function

That's why the capitalization is different